components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -1420,6 +1420,19 @@ public static void startSuperTenantFlow() {
         carbonContext.setTenantDomain(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME);
     }
 
+    /**
+     * This method will start a tenant flow.
+     */
+    public static void startTenantFlow() {
```

Comment:
We should be able to remove this method.

Reply:
Fixed